### PR TITLE
Email GUI

### LINKED ISSUE #992: GUI Email Sender Application
# Aim
This is an e-mail sending GUI application.
# Details
In this application, I will be building a python based GUI application that will be able to send E-mail.

### PR DESCRIPTION
**Pull request by Amit366**
# Description

Added Email sender gui script

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] I have added the project meta data in the PR template.
- [x] I have created the ``requirements.txt`` file if needed.

## Project Metadata

`` If there is no-file/nothing to fill the below fields with, then type: none ``

`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.

Category:
- [ ] Calculators
- [ ] AI/ML
- [ ] Scrappers
- [ ] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [x] Others

Title: Email sender GUI

Folder: Email GUI

Requirements: none

Script: script.py

Arguments: none

Contributor: Amit366

Description: An email sender Gui to send emails

### CODE REVIEW

File: Email GUI/script.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    sender_info = sender_address.get()
+
+    password_info = password.get()
+    
+    #print(address_info,email_body_info)
+    
+    #sender_email = input("Enter sender's email: ") 
+    
+    #sender_password = input("Enter password: ")
```
Comment (kaustubhgupta) [inline Email GUI/script.py:19]: you can remove these comments

File: Email GUI/script.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    print("Login successful")
+    
+    server.sendmail(sender_info,address_info,email_body_info)
+    
+    print("Message sent")
+    
+    address_entry.delete(0,END)
+    email_body_entry.delete(0,END)
+    password_entry.delete(0,END)
+    sender_address_entry(0,END)
```
Comment (kaustubhgupta) [inline Email GUI/script.py:36]: This is not defined between the line 1 to 35 and that's why the pylint is showing error

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
you have force pushed the branch and that's why other commits are present in this PR. Kindly fix it. For future ref, don't force push branch until extremely imp

**Comment by Amit366** [comment]
@kaustubhgupta I have removed the extra files

File: Email GUI/Readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    Run the command: python script.py/python3 script.py 🧐
+
+### Step 4:
+
+    Sit back and Relax. Let the Script do the Job. ☕
+
+### Requirements
+
+    - smtplib
+    - tkinter
```
Comment (kaustubhgupta) [inline Email GUI/Readme.md:32]:
```suggestion
- smtplib
- tkinter
```

File: Email GUI/Readme.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    
+## Author
+   
+   Amit Kumar Mishra
```
Comment (kaustubhgupta) [inline Email GUI/Readme.md:36]:
```suggestion
[Amit Kumar Mishra](https://github.com/Amit366)
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Just a readme update :)

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍

**Review by santushtisharma10 — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Pull request merged**